collect_app/src/main/java/org/odk/collect/android/geo/GoogleMapFragment.java

```diff
@@ -635,8 +640,7 @@ private float getIconAnchorValueY(@IconAnchor String iconAnchor) {
     }
 
     private BitmapDescriptor getBitmapDescriptor(int drawableId) {
-        return BitmapDescriptorFactory.fromBitmap(
-            IconUtils.getBitmap(getActivity(), drawableId));
+        return BitmapDescriptorFactory.fromBitmap(MapMarkerCache.getMarkerBitmap(drawableId, getContext()));
```

Comment:
This is used only in `GoogleMapFragment` because other providers do not require bitmaps.